src/js/components/tasks/table.tsx

```diff
@@ -178,6 +196,8 @@ const AssigneeTableCell = ({
         assignedUser = item.assigned_qa;
         break;
     }
+    const currentUserSelected =
+      assigneeSelection?.login === currentUser.username;
```

Comment:
Now that I think on it, this should probably be defined in the Modal to do checks on whether the assignee is the current user.